MAUI/TreeMap/brush-settings.md

```diff
@@ -0,0 +1,438 @@
+---
+layout: post
+title: Brush settings in .NET MAUI TreeMap (SfTreeMap) control | Syncfusion
+description: Learn here how to apply brush settings for tree map items in Syncfusion .NET MAUI TreeMap control, enhancing the tree map visualization.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# Leaf item brush settings in .NET MAUI TreeMap (SfTreeMap)
+
+The brush settings is used to customize the fill colors for leaf items based on ranges or values, offering four brush settings: `UniformBrushSettings,` `RangeBrushSettings,` `DesaturationBrushSettings,` and `PaletteBrushSettings.` Each setting provides unique options for defining and applying color schemes, enhancing the visualization of the tree map.
```

Comment:
The brush settings are used to customize the fill colors for leaf items based on ranges or values, offering four brush settings: `UniformBrushSettings,` `RangeBrushSettings,` `DesaturationBrushSettings,` and `PaletteBrushSettings.` Each setting provides unique options for defining and applying color schemes, enhancing the visualization of the treemap.

Reply:
Updated.